.gitlab-ci.yml

```diff
@@ -150,6 +148,19 @@ test-dependency-rules:
   script:
     - .maintain/ensure-deps.sh
 
+test-prometheus-alerting-rules:
+  stage:                           check
+  image:                           paritytech/tools:latest
+  <<:                              *kubernetes-build
+  rules:
+    - if: $CI_COMMIT_BRANCH
```

Comment:
`test-prometheus-alerting-rules` should run on pull requests as well!

Reply:
it [does](https://gitlab.parity.io/parity/substrate/-/pipelines/119948), this `$CI_COMMIT_BRANCH` in our case is a wildcard for "in any case". I would avoid it if it would be possible to mention just `changes:`.